ports/unix/variants/longlong/mpconfigvariant.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Damien P. George
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+// This config exists to test that the MICROPY_LONGINT_IMPL_LONGLONG variant
+// (i.e. minimal form of "big integer" that's backed by 64-bit int only) builds
+// and passes tests.
+
+#define MICROPY_LONGINT_IMPL           (MICROPY_LONGINT_IMPL_LONGLONG)
+
```

Comment:
@yoctopuce suggested we test `MICROPY_OBJ_REPR_C` and `MICROPY_LONGINT_IMPL_LONGLONG` together, which makes sense - and I don't think we have CI coverage for the former at the moment.

@dpgeorge what do you think?

Reply:
Yes, that's a very good idea to test `MICROPY_OBJ_REPR_C` in CI.

It's enabled on esp8266 and stm32 Arduino boards, so it does get tested for a release.  But would be great to test it in CI.

Enabling representation C in this new longlong variant is a good idea!

Reply:
`MICROPY_OBJ_REPR_C` seems to need more work than just enabling it for this config, so I'll look at that for a follow-up PR.